daemon/mount.c

```diff
@@ -239,6 +241,21 @@ mount_entry_get_mount_data(const mount_entry_t *mntent)
 	return mntent->mount_data;
 }
 
+char *
+mount_entry_get_verity_sha256(const mount_entry_t *mntent)
+{
+	ASSERT(mntent);
+	return mntent->verity_sha256;
+}
+
+void
+mount_entry_set_verity_sha256(mount_entry_t *mntent, char *sha256)
+{
```

Comment:
Do we need an ASSERT(sha256) here as well?

Reply:
Added IF_NULL_RETURN(sha256);